package.json

```diff
@@ -68,7 +68,7 @@
     "eslint-plugin-flowtype": "^3.2.0",
     "eslint-plugin-react": "~6.7.0",
     "flow-bin": "^0.80.0",
-    "gl": "^4.0.3",
+    "gl": "4.1.1",
```

Comment:
- Why lock the version?
- I don't believe this dependency is needed any more. Just remove it?